content/04-r-rstudio/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "R & Rstudio"
+title = "Cloning and Collaborating with R & Rstudio"
```

Comment:
I got rid of 'cloning and collaborating with' from the other chapters, do you want this in?

Reply:
That's just the chapter landing page. All chapters require one. It's not got any info in it

Reply:
Oh sorry!! Completely misread that comment!